source/components/index.md

```diff
@@ -1,19 +1,29 @@
 # Harnesses, probes, and detectors
```

Comment:
Scenarios are added now and putting that in the title is too long in the sidebar. Maybe we can change this to something shorter and general?

Reply:
"Evaluation structure" ?

Reply:
"Evaluation components" ?